generator/Cargo.toml

```diff
@@ -5,15 +5,15 @@ authors = ["Maik Klein <maikklein@googlemail.com>"]
 edition = "2018"
 
 [dependencies]
-bindgen = "0.58"
-heck = "0.3"
+bindgen = "0.59.2"
+heck = "0.4.0"
 itertools = "0.10"
 nom = "6.0"
-once_cell = "1.7"
+once_cell = "1.9.0"
```

Comment:
Again no need for patch versions.

Reply:
that's a bug in cargo upgrade --skip-compatible

Reply:
I'd do these by hand :)